libs/client/cornflow_client/constants.py

```diff
@@ -107,6 +107,101 @@
     pl.LpStatusUndefined: STATUS_UNDEFINED,
 }
 
```

Comment:
it may be nice to add a readme with all cornflow parameters names and definitions

Reply:
That will be done latter, I do not want to keep this more time open.